rl755/models/common/lstm.py

```diff
@@ -0,0 +1,29 @@
+import tensorflow as tf
+from rl755.models.common.sequential import SequentialModel
+
+class LSTM(SequentialModel):
+    def __init__(self, hidden_size, output_size, **kwargs):
+        super().__init__(**kwargs)
+        self.output_size = output_size
+        self.hidden_size = hidden_size
+        self.lstm = tf.keras.Sequential([
+            tf.keras.layers.Dense(units=self.hidden_size, activation=None),
```

Comment:
Do we need the dense layer here? I'm unfamiliar with LSTMs, so feel free to leave it in if its here for a reason.

Reply:
It's there not for some specific reasons, just based on some examples I saw online. I think I will delete it.